metals/src/main/scala/scala/meta/internal/parsing/Trees.scala

```diff
@@ -114,6 +114,6 @@ object Trees {
   /* Tokenizing works perfectly fine with 212 dialect as long as we are only
    * interested in having stable results. This is not the case for parsing.
    */
-  val defaultTokenizerDialect = scala.meta.dialects.Scala213
+  val defaultTokenizerDialect: Dialect = scala.meta.dialects.Scala213
```

Comment:
We should probably remove it if possible.

Reply:
Yea, I was thinking the same thing. Lemme see if I can tackle the rest of the places that use this as well.

Reply:
Or really `scala.meta.dialects.Scala3` would probably work? We can make it a custom dialect, but might be simpler than in case of ScalaMtags

Reply:
🤔 so this is proving to be a bit harder than I thought since all the digests would need this as well. So it's not ideal to have to pass around `Trees` to them. I think it might be better to fix this here, and then to create a follow-up to figure out how to fully get rid of them.

Reply:
We don't really need Trees for digests. Sbt will stay on Scala 2 for a long while, I think the same with Mill.

Reply:
Ah then honestly the main other things that are using it then are bench tests and digests. So it shouldb be good then.